packages/desktop-gui/src/lib/utils.js

```diff
@@ -84,4 +84,32 @@ module.exports = {
     // remove leading 'cypress/integration' from spec
     return spec.replace(cyDirRegex, '')
   },
+
+  stripSharedDirsFromDir2 (dir1, dir2, osName) {
```

Comment:
I think it would be good to have some unit tests for this method.

Reply:
ugh, yah....we do have unit tests in the Dashboard for this method, so I just copied it over. The duplicated code is such a pain. I will copy the unit tests over also.